verifier/src/main/AndroidManifest.xml

```diff
@@ -38,6 +40,7 @@
 
 		<activity
```

Comment:
Could also add `exported="false"`, since the onboarding activity is only referenced internally